debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -301,6 +301,14 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
                     + "In '" + SnapshotIsolationMode.READ_UNCOMMITTED.getValue()
                     + "' mode neither table nor row-level locks are acquired, but connector does not guarantee snapshot consistency.");
 
+    public static final Field TASK_DATABASE_NAMES = Field.create("task.database.dbnames")
```

Comment:
So we have `DATABASE_NAMES` and `TASK_DATABASE_NAMES`. Both are required, both come from `SqlServer` config ... If I get it right `TASK_DATABASE_NAMES` should be optional and private. Alternatively we can set subset of `DATABASE_NAMES` back to `DATABASE_NAMES`. @morozov What was the plan?

Reply:
one thing we can maybe do is make it an internal field and private.

Reply:
We need both but `TASK_DATABASE_NAMES` should be defined at the task level, not at the connector level. We agreed to postpone the cleanup and leave it defined here.

Reply:
It's not clear what the point in passing 2 configs to partition provider:
```java
final SqlServerConnectorConfig connectorConfig = new SqlServerConnectorConfig(config);
//...
new SqlServerTaskPartition.Provider(connectorConfig, config, metadataConnection).getPartitions()
```
I'd rename `connectorConfig` to `taskConfig` to simplify things.

Reply:
I created a new ticket that makes task config an actual configuration class (https://sugarcrm.atlassian.net/browse/CXP-948) I think that would clean up some of the confusion.